utility/src/main/java/org/oppia/util/firebase/CrashlyticsWrapper.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Controller for providing custom crash reporting to Firebase Crashlytics */
+@Singleton
+open class CrashlyticsWrapper @Inject constructor(
+  private var firebaseCrashlytics: FirebaseCrashlytics
+) {
+
+  /** Logs a custom non-fatal exception to Firebase Crashlytics */
+  open fun logException(exception: Exception) {
+      firebaseCrashlytics.recordException(exception)
+  }
+
+  /** Logs a custom log message which can be put alongside a crash report to Firebase Crashlytics */
+  open fun logMessage(message: String) {
```

Comment:
When will this be used? It doesn't seem to be in this PR.

Reply:
Made for future usage (if any). We can probably use it to provide a better insight in our crash reports.